src/api_authentication_app_skeleton/src/models/user_token.cr

```diff
@@ -13,8 +13,10 @@ class UserToken
     JWT.encode(payload, Lucky::Server.settings.secret_key_base, ALGORITHM)
   end
 
+  # Both the payload and header are provided from the JWT decode method
+  # Should you require the header, modify the `_` variable below to something like `header`
   def self.decode_user_id(token : String) : Int64?
-    payload, _header = JWT.decode(token, Lucky::Server.settings.secret_key_base, ALGORITHM)
+    payload, _ = JWT.decode(token, Lucky::Server.settings.secret_key_base, ALGORITHM)
```

Comment:
I'm pretty against `_` because it is so unclear it needs a comment. I wonder if we can disable this rule in Ameba? I'd say for me this is a dealbreaker. Probably quite pedantic, but I think it is helpful to be able to label unused variables so you don't need comments.

Reply:
Totally agreed, @paulcsmith. The only trouble is that unless we *also* disabled this in a default-included `.ameba.yml` file, every newly-generated Lucky app will still produce an Ameba error by default.

@jwoertink and I talked about this a bit in Gitter, and maybe it's worth leaving this PR on hold until I open and resolve an issue with the Ameba team directly.

IMO, there's no reason that you *wouldn't* want to show what an unused variable actually is in case you need it in the future.

Reply:
The newest version of Ameba is cool with this, so it's been reverted in this PR!